azure-pipelines.yml

```diff
@@ -0,0 +1,21 @@
+# Ref: https://aka.ms/yaml
+
+jobs:
+- job: test
+  pool:
+    vmImage: 'Ubuntu 16.04'
```

Comment:
Update: It turns out, there's no 18.04 image, but [it's next in line for them](https://github.com/Microsoft/azure-pipelines-image-generation/issues/506), so hopefully this can be updated in a few months.

Reply:
Pool image is actually not related to our environment. `vmImage` makes sure pipeline is working on Linux environment (with Azure's Ubuntu 16.04 vm image). It will always build and use our own development image with docker.